src/serializer.js

```diff
@@ -14,12 +14,23 @@ function createSerializer(styleSheet) {
     val.withStyles = true
     const printedVal = printer(val)
     if (styles) {
-      return `${styles}\n\n${printedVal}`
+      return replaceClassNames(selectors, styles, printedVal)
     } else {
       return printedVal
     }
   }
 
+  function replaceClassNames(selectors, styles, code) {
+    let index = 0
+    return selectors.reduce((acc, className) => {
+      const escapedRegex = new RegExp(
```

Comment:
I'd put it in `replace-class-names.js` (I prefer to avoid the file casing inconsistencies of operating systems by using kebab casing for all filenames).